public/index.html

```diff
@@ -75,5 +73,6 @@
       To begin the development, run `npm start` or `yarn start`.
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
-  </body>
-</html>
+</body>
+
+</html>
```

Comment:
It is a good practice to add space at the end of file.

Reply:
ok, noted